### PR TITLE
Rename Machine group to Deployment group - Stage 1

### PR DESCRIPTION
**Pull request by chshrikh**
Rename Machine group to Deployment group - Stage 1 - Support deploymentgroup flag along with existing machinegroup flag

### CODE REVIEW

**Comment by KrishnaAdityaB** [comment]
@chshrikh 
you needed to maintain both the flags for sometime because of the script generation in VSTS. The generated script should work with both versions of agent - until VSTS starts pointing to the latest one. Right?

**Comment by chshrikh** [comment]
yes not only for the script in VSTS but for Azure vm extension as well ( Ibiza extension )

File: src/Agent.Listener/CommandSettings.cs [hunk 1]
```diff
@@ -62,7 +63,7 @@ public sealed class CommandSettings
         public bool Help => TestFlag(Constants.Agent.CommandLine.Flags.Help);
         public bool Unattended => TestFlag(Constants.Agent.CommandLine.Flags.Unattended);
         public bool Version => TestFlag(Constants.Agent.CommandLine.Flags.Version);
-        public bool MachineGroup => TestFlag(Constants.Agent.CommandLine.Flags.MachineGroup);
+        public bool MachineGroup => TestFlag(Constants.Agent.CommandLine.Flags.MachineGroup) || TestFlag(Constants.Agent.CommandLine.Flags.DeploymentGroup);
```
Comment (TingluoHuang) [inline src/Agent.Listener/CommandSettings.cs]: Chnage bool MachineGroup to DeploymentGroup? since DeploymentGroup will be the final name.

Comment (chshrikh) [inline src/Agent.Listener/CommandSettings.cs]: I was planning to change variable, method names as part of my final stage of this story

**Pull request merged**